### PR TITLE
Document default install directory

### LINKED ISSUE #17064: dotnet-install-script.md too windows centric
https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-install-script.md documents the default installation directory only for windows, and not for linux.

Edited to add Document Details for linking.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a2e484a9-5e0c-eb3e-1413-7b507c4fa07d
* Version Independent ID: 5bc7caa5-f1c2-d7d5-fe02-80c65cca9672
* Content: [dotnet-install scripts - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#feedback)
* Content Source: [docs/core/tools/dotnet-install-script.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-install-script.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #17064

### CODE REVIEW

File: docs/core/tools/dotnet-install-script.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ You can install a specific version using the `-Version|--version` argument. The
 
 - **`-InstallDir|--install-dir <DIRECTORY>`**
 
-  Specifies the installation path. The directory is created if it doesn't exist. The default value is *%LocalAppData%\Microsoft\dotnet*. Binaries are placed directly in this directory.
+  Specifies the installation path. The directory is created if it doesn't exist. The default value is *%LocalAppData%\Microsoft\dotnet* on Windows and */usr/share/dotnet* on Linux. Binaries are placed directly in this directory.
```
Comment (Thraka) [inline docs/core/tools/dotnet-install-script.md:95]: Is there a common variable we can use in this like `LocalAppData` is for Windows? I think that your linux path is actually a macos path. My linux machine is dying and I need to refresh to double check.

Comment (tdykstra) [reply docs/core/tools/dotnet-install-script.md:95]: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet#environment-variables doesn't mention such a variable:

> The default location on Linux and macOS is `/usr/share/dotnet`.

Comment (Thraka) [reply docs/core/tools/dotnet-install-script.md:95]: yes. I'm unsure why I had a different path. I had asked someone back before I got a mac to check with.

Comment (Thraka) [inline docs/core/tools/dotnet-install-script.md:95]:
```suggestion
  Specifies the installation path. The directory is created if it doesn't exist. The default value is *%LocalAppData%\Microsoft\dotnet* on Windows and */usr/share/dotnet* on Linux/macOS. Binaries are placed directly in this directory.
```

**Review by Thraka — APPROVED** [review]
Approved with a small suggestion.

**Pull request merged**